packages/react-scripts/template/README.md

```diff
@@ -771,6 +772,83 @@ Please refer to these two threads for reference:
 
 Create React App will add decorator support when the specification advances to a stable stage.
 
+## Integrating with Electron
+
+[Electron](https://electron.atom.io/) offers an API to create cross platform desktop applications using HTML, CSS and JavaScript in a Node environment. Electron can render a Create React App with minimal changes.
+
+Start by creating a React app using `create-react-app` command line tool. Then, install the following packages:
+- `electron` is the Electron package that includes all necessary modules to create an Electron app.
+- `electron-packager` is a command line tool necessary to bundle an Electron app.
+- `electron-is-dev` is a simple package that when required returns a boolean that can be used to differentiate dev from bundled Electron apps.
+- `concurrently` is a command line tool that runs two processes concurrently.
+- `wait-on` is a command line tool that waits for a file, port, socket, or http resource to be available.
+
+At the root of the React application folder you will need to create an Electron script for initializing your Electron app. [Electron's quick start](https://github.com/electron/electron-quick-start) `main.js` is a good starting point. Extra logic needs to be added to the window creation in order to render the React application in both development and bundled environments.
+
+```javascript
+// Dev flag
+const electronIsDev = require('electron-is-dev');
+// Create the browser window.
+mainWindow = new BrowserWindow({width: 800, height: 600})
+
+// and load the index.html of the app.
+mainWindow.loadURL(
+  electronIsDev
+    ? 'http://localhost:3000' // Dev server ran by react-scripts
+    : `file://${path.join(__dirname, '/build/index.html')}` // Bundled application
+);
+```
+
+Once this file is created, the `package.json` needs to be edited:
+- Set the Electron script (here `main.js`) as `main`.
+- Set application's root (`./`) as `homepage`.
```

Comment:
Just curious: What does this and why is it needed?

Reply:
so that the CRA's production build will resolve index.html's resources path to be related to the current index.html rather than server'root 

https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#serving-the-same-build-from-different-paths